### PR TITLE
Support iOS 13 Dark Mode

### LINKED ISSUE #424: not support dark mode on iOS 13
Hello everyone ,

I updated all my devices and my applications for iOS 13.
I work only light content mode for my application. "Actionsheetpicker" does not support if my phone is in dark mode

can u help with this?

### PR DESCRIPTION
**Pull request by arnoldxt**
support dynamic background color in iOS 13

Will close #424

### CODE REVIEW

**Comment by koust** [comment]
I tried and it didn't work

**Comment by plbeaudoin** [comment]
It works for me. This what I'm using in Trip Planner for Caltrain.

**Comment by skywinder** [comment]
Thanks, @arnoldxt  and all for participation. 
Looks good.
I'm currently don't have a chance to debug it by myself. 
I would very appreciate for code review and test this update. 
---
btw, 
@plbeaudoin are you the developer of Caltrain?

I was pretty amazed when I saw my tool in the app during the trip in US! 👍

**Comment by plbeaudoin** [comment]
I am not the original app author but we are indeed still using it 👍🏼

**Comment by koust** [comment]
it works. I'm really happy right now :)
Thanks, @arnoldxt and all for participation.

**Comment by ChenPingSheng** [comment]
should add "#ifdef __IPHONE_13_0" macro to support Xcode10 or lower version.

**Comment by skywinder** [comment]
The current  doesn’t support it, really?
Looks like predicate should work fine. I can test in on the next week.

**Comment by arnoldxt** [comment]
#ifdef detects if your app support iOS 13 at compile time only
It cannot detect the app is running on iOS 13 or not at running time

**Comment by ChenPingSheng** [comment]
> #ifdef detects if your app support iOS 13 at compile time only
> It cannot detect the app is running on iOS 13 or not at running time

For some reason, I need use Xcode 10 to build app, so I hope your merge can do some compatible job to support Xcode 10. That's will be better, just a suggestion.

**Comment by goooogler** [comment]
This works for me.  Thank you.

**Comment by jinsnet** [comment]
This works for me. thanks

**Comment by skywinder** [comment]
@ChenPingSheng would you mind to make PR with support of Xcode 10 and below?

**Comment by skywinder** [comment]
I prefer to support old versions of xcode/ios if it's not hard to do. so let's giva a try.


Usually, I solve versions via 
```
if (NSFoundationVersionNumber > NSFoundationVersionNumber_iOS_6_1) 
{
}
``` 

block. But it's stops to update after `NSFoundationVersionNumber_iOS_9_x_Max`

I used it according apple's guide. 

here is details https://stackoverflow.com/a/18076628/1698467

ANy suggestions, how to do this?
@arnoldxt @ChenPingSheng > thoughts?

**Comment by ChenPingSheng** [comment]
How about this:
`#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000`

**Comment by skywinder** [comment]
@ChenPingSheng GOod point. I need to fix it. But still looks like temporary option. Probably we have to migrate to new xcode and that's it.

File: Pickers/AbstractActionSheetPicker.m [hunk 1]
```diff
@@ -540,7 +540,12 @@ - (UIBarButtonItem *)createToolbarLabelWithTitle:(NSString *)aTitle
         textSize = toolBarItemLabel.attributedText.size;
     }
     else {
-        [toolBarItemLabel setTextColor:(NSFoundationVersionNumber > NSFoundationVersionNumber_iOS_6_1) ? [UIColor blackColor] : [UIColor whiteColor]];
+        if (@available(iOS 13.0, *)) { // Support iOS 13 system dark mode
```
Comment (skywinder) [inline Pickers/AbstractActionSheetPicker.m]: Please wrap it in as it looks there https://github.com/openid/AppAuth-iOS/pull/412/files#diff-2b222704dbfd09d494d35a28706e0f3fR30-R36

` #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 `

**Comment by skywinder** [comment]
Thank you for the support.  Merged!

**Pull request merged**